frappe/__init__.py

```diff
@@ -1364,7 +1363,7 @@ def get_active_domains():
 
 	return active_domains
 
-def get_system_country():
-	if local.system_country is None:
-		local.system_country = db.get_single_value('System Settings', 'country') or ''
-	return local.system_country
+def get_system_settings(key):
```

Comment:
@rmehta 
Fixed. I have added `local.system_settings = _dict()` in the init and then added the key value pairs. Passing parameter is better and generic approach